src/applications/simple-forms/20-10207/config/submit-transformer.js

```diff
@@ -82,7 +82,7 @@ export default function transformForSubmit(formConfig, form) {
   ) {
     const { street, street2, city } = transformedData.veteranMailingAddress;
     transformedData.veteranMailingAddress.street = street.slice(0, 30);
-    transformedData.veteranMailingAddress.street2 = street2.slice(0, 5);
+    transformedData.veteranMailingAddress.street2 = street2?.slice(0, 5);
```

Comment:
Is it ok that I inserted this `?` here? I was getting console errors with a missing `street2`.

Reply:
seems fine to me